blazor/predefined-dialogs/draggable.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Draggable in Blazor Predefined Dialogs Component | Syncfusion
+description: Checkout and learn here all about draggable in Syncfusion Blazor Predefined Dialogs component and much more details.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
```

Comment:
Documentation: UG

Reply:
In `Breadcrumb` component they have given lower case only. So not corrected this changes.

<img width="662" alt="image" src="https://user-images.githubusercontent.com/93525217/182564491-6280ce19-93a0-4997-94d2-e4708dae4964.png">